her-tech-trail/Trail-backend
# #1 Setup Project Structure

#### Description of Task to be completed?
- add basic project folders
- setup test
- setup babel
- create a server entry file
- add a basic test

 #### How should this be manually tested?
- clone the repo with `git clone <branch name>`
- run `npm install` to install the needed dependencies
- run `npm run start:dev` to start the dev server
- navigate your browser to `localhost:5000/api`
- you should now see a welcome message

 #### Any background context you want to provide?
- ensure you have node installed in your computer
-ensure you have internet access

 #### What are the relevant story description?
- Setup project structure

 #### Screenshots (if appropriate)
- none

 #### Questions:
- none
